plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF

```diff
@@ -8,6 +8,7 @@ Bundle-Localization: plugin
 Eclipse-BundleShape: dir
 Eclipse-BuddyPolicy: registered,
  ext
+Eclipse-RegisterBuddy: org.apache.log4j

```

Comment:
Would it help to define another bundle to separate concerns, in this case logging with log4j to e.g. org.locationtech.udig.log4j ? So it might be easier for potential developers to use another logging framework rather than log4j. 
Other could contribute a commons.logging adapter or even a slf4j logging adapter as well.

Reply:
I feel like if somebody wants to do this they can. I'm not sure there is a reason for us to do it now.

Reply:
I had a look at it, please see pull request https://github.com/egouge/udig-platform/pull/7

@egouge much appreciate if you could have a look at and test in your environment (SMART??)

Reply:
I merged this and it seems to work as expected